changelogs/fragments/ansible_test_rmv_macos_132.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  - ansible-test - Removed test remotes: macos/13.2
```

This is now a list with a single-element dictionary.
```suggestion
  - "ansible-test - Removed test remotes: macos/13.2"
```

---

Ah, right,thanks!